apps/condo/domains/meter/utils/metersImporters/AbstractMetersImporter.ts

```diff
@@ -149,12 +149,19 @@ export abstract class AbstractMetersImporter {
                         } else {
                             const mutationError = errors[errorIndex++]
 
-                            const messageForUser = get(mutationError, ['extensions', 'messageForUser'])
+                            const messageForUser = get(
+                                mutationError,
+                                ['extensions', 'messageForUser'], // Error thrown from the mutation itself
+                                get( // Error thrown by models created within the mutation
+                                    mutationError,
+                                    ['originalError', 'errors', 0, 'extensions', 'messageForUser'],
```

Comment:
Is it possible to provide 'fallback' error message in case of no `messageForUser` provided?

Reply:
Maybe, but this is temporary code. For now we moving this logic to backend. Error processing will be changed.